api/handlers/userhandler.py

```diff
@@ -100,6 +105,19 @@ def post(self):
     def _init_storage(self):
         self.storage = containerstorage.ContainerStorage('users', use_object_id=False)
 
+    def _cleanup_user_permissions(self, uid):
+        try:
+            config.db.collections.delete_many({'curator': uid})
```

Comment:
This makes me nervous. I don't like the idea of shares evaporating, especially if others are using them. Had a conversation with @elisapoquette on how we want to proceed; I'll let her comment from here